Imgui/CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ elseif(PLATFORM_UNIVERSAL_WINDOWS)
     list(APPEND SOURCE src/ImGuiImplUWP.cpp)
     list(APPEND INTERFACE interface/ImGuiImplUWP.hpp)
 elseif(PLATFORM_LINUX)
+    if(PLATFORM_FREEBSD)
+        if(EXISTS /usr/local/include)
+            include_directories(/usr/local/include)
+        endif()
+    endif()
```

Comment:
This code is copy-pasted a dozen of times. Common build settings need to be set up through `Diligent-BuildSettings` or `Diligent-PrivateBuildSettings`

Reply:
Aight, I can do that once I get home.